lib/templates/android_screen_base.tt

```diff
@@ -183,4 +183,23 @@ class AndroidScreenBase < Calabash::ABase
     # Clicking in the Done button
     touch "* id:'button1'"
   end
+  
+ 
+  
+  # @option [String] id The view's id (optional)
+  # @option [String] The complete view query
+  # @return [String] returns a string associated with a single view, if the case
+  # @return [Array] return an array[String] associated with the views, if the case
+  # Example ->     get_text_from(query_string_variable)
+  def get_text_from(id = nil, query)
+    query_text = "* id:'#{id}'" unless id.nil?
+    query_text = query unless query.nil?
+    array_text = query(query_text, :text)
+
+    # returns an array[String] or text dependending of the quantity of views found
+    return array_text.first unless array_text.size > 1
```

Comment:
@oscartanner ,
- I'm gonna delete the _get_ from the name method;
- I think your two methods separation makes sense and I'm gonna change it.